src/test/test-dlopen-so.c

```diff
@@ -44,6 +45,10 @@ static int run(int argc, char **argv) {
         assert_se(dlopen_libfido2() >= 0);
 #endif
 
+#if HAVE_BPF
```

Comment:
Looks like a typo. Did you mean `HAVE_LIBBPF`? 

There are two BPF-related macro: `HAVE_LIBBPF` and `BPF_FRAMEWORK`.
I can't see how `HAVE_BPF` is defined.

Reply:
oops good catch, the test passed so didn't notice